### DISCUSSION TITLE
Using Livewire to populate form input after wire:click

### DISCUSSION THREAD

**Discussion by jason-rehatched** [livewire/livewire#3522] [1. Help] [answered]
Hello All,

Thank you for taking the time to read and perhaps reply. I think I may have gotten myself so far down the rabbit hole, I'm overlooking an obvious answer.  

Here is what I'm trying to accomplish:

I have a form with numerous text inputs rendered using Laravel Collective. I have a Bootstrap modal which allows a user to input a URL:

```
<div>
    <input wire:model="url" type="text">
    <button class="btn btn-primary" wire:click="scrape">Scrape</button>
</div>
```

 Upon submitting the URL, I have a Livewire method which parses specific data from the URL:

```
`public $url;

public function scrape()
    {
      $link = GoutteFacade::request('GET', $request->url);

      //dd($link->filter('title')->text());
    }`
```

What I'm trying to do is pass $link to my Livewire component (or Blade file if need be) so I can populate an input field like so:

```
`<div class="form-group">
                                {!! Form::label('title', 'Title', ['class' => 'control-label']) !!}
                                {!! Form::text('title', (isset($link) ? $link->filter('title')->text() : null), ['class' => 'form-control', 'placeholder' => 'Your catchy title']) !!}
                                @error('title')
                                <x-alert type="danger" :message="$message" />
                                @enderror
                            </div>`
```

I'm trying to avoid doing a complete page reload, hence Livewire to provide a better UX. Just cannot wrap my head around this.

Thank you in advance!

**Comment by ryatkins** [comment, marked as answer]
@jason-rehatched - give this example I made a look
https://laravelplayground.com/#/snippets/50d57d27-e69e-4121-9513-0c6259f5c3a9

**Reply by jason-rehatched** [reply]
THANK YOU!!! As I thought, I was over-thinking it.  Thank you once again for your time and help!
